go/cmd/vtbackup/vtbackup.go

```diff
@@ -349,6 +347,16 @@ func takeBackup(ctx context.Context, topoServer *topo.Server, backupStorage back
 		return fmt.Errorf("can't restore from backup: %v", err)
 	}
 
+	// Disable redo logging (if we can) before we start replication.
+	disabledRedoLog := false
+	if mysqld.CanDisableRedoLog() {
+		if err := mysqld.DisableRedoLog(ctx); err != nil {
```

Comment:
What is the goal of this PR? Seems that it is to allow a just restored MySQL instance (by vtbackup) to catch back up with the shard replication position more quickly so that the larger shard backup operation finishes more quickly? I believe so, but had to learn more about vtbackup for this to be clear. :-)

Reply:
You sensed the goal correctly. Sorry for not calling this out in the PR description. Have added a `Details` section in the PR description to make this more clear.